demo/kubernetes/README.md

```diff
@@ -71,6 +71,7 @@ At this point, you can follow the steps in the rook [quickstart](/README.md#bloc
 To use a block device in the kubernetes cluster you will first need to create a block image for that device.  **In your rook-client pod** from above, run:
 ```
 rook block create --name demoblock --size 1073741824
```

Comment:
what if we put this rook command in the previous section so it's obvious that you should be running it in the pod, then you exit at the end of the section?